transformations/p1_noun_transformation/test.json

```diff
@@ -0,0 +1,71 @@
+{
+  "type": "add_noun_definition",
+  "test_cases": [
+    {
+      "class": "AddNounDefinition",
+      "inputs": {
+        "sentence": "Andrew finally returned the Comic book to Chris that I bought last week"
+      },
+      "outputs": [
+        {
+          "sentence": "andrew finally returned the comic book(publication of comics art) to chris that i bought last week(time unit equal to seven days)"
+        }
+      ]
+    },
+    {
+      "class": "AddNounDefinition",
+      "inputs": {
+        "sentence": "Chris borrowed the Comic book from Andrew last weekend"
+      },
+      "outputs": [
+        {
+          "sentence": "chris borrowed the comic book(publication of comics art) from andrew last weekend"
+        }
+      ]
+    },
+    {
+      "class": "AddNounDefinition",
+      "inputs": {
+        "sentence": "Turn off the light please"
+      },
+      "outputs": [
+        {
+          "sentence": "turn off the light please(english word indicating politeness)"
+        }
+      ]
+    },
+    {
+      "class": "AddNounDefinition",
+      "inputs": {
+        "sentence": "I love cat"
+      },
+      "outputs": [
+        {
+          "sentence": "i love cat(domesticated felid species)"
+        }
+      ]
+    },
+    {
+      "class": "AddNounDefinition",
+      "inputs": {
+        "sentence": "Thomas loves to cook chicken every Monday"
+      },
+      "outputs": [
+        {
+          "sentence": "thomas loves to cook chicken(subspecies of domesticated bird) every monday"
```

Comment:
You should mention about this as a limitation - the right sense is important to be picked up, isn't it?